packages/runner-ct/src/main.tsx

```diff
@@ -32,7 +32,9 @@ const Runner: any = {
   },
 
   start (el, base64Config) {
-    const ws = createWebsocket()
+    const config = window.UnifiedRunner.decodeBase64(base64Config)
```

Comment:
this appears to be parsed here and then also parsed on line 52, can it be updated to only parse once?

Reply:
Fixed this: https://github.com/cypress-io/cypress/commit/612bf22d3fa1ec9cb836e0bf15d2b623728336aa